bin/up

```diff
@@ -8,4 +8,4 @@ mutagen daemon start
 echo -e ${prefix}The mutagen daemon will stay running until you stop it manually with \`mutagen daemon stop\`.$suffix
 
 echo -e ${prefix}Running mutagen-compose...$suffix
-mutagen-compose -f "$basedir"/docker-compose.yml up --build --detach
+PRIVATE_NETWORK_IP=$(ifconfig -l | xargs -n1 ipconfig getifaddr) mutagen-compose -f "$basedir"/docker-compose.yml up --build --detach
```

Comment:
I guess `$(ifconfig -l | xargs -n1 ipconfig getifaddr)` might be Mac specific?

`-l` option of `ifconfig` seems to be UNIX specifc(?). On my Ubuntu (WSL2) `-l` option is not found. And if I'm right `ipconfig` is Windows (and Mac?) command. It is not found on my Ubuntu.

Requesting users to rewrite this line depending on platform might be ok. But at least noted in the document or somewhere else.